docs/skills_guide.md

```
@@ -0,0 +1,250 @@
+- [Introduction](#introduction)
+- [Configuration](#configuration)
+  - [Option A: Directory of Skills](#option-a-directory-of-skills)
+  - [Option B: Individual Skill Paths](#option-b-individual-skill-paths)
+- [Skill Directory Structure](#skill-directory-structure)
+- [SKILL.md Format](#skillmd-format)
+  - [Frontmatter Fields](#frontmatter-fields)
+  - [Body Content](#body-content)
+- [Creating a Skill](#creating-a-skill)
+- [How Skills Work at Runtime](#how-skills-work-at-runtime)
+- [Limitations](#limitations)
+- [Error Reference](#error-reference)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)